### ISSUE TITLE
Issue in Importing Libraires 

### ISSUE THREAD

**Issue by AJSTYLE-lab** [nicknochnack/ImageClassification#7]
The error is in installing TensorFlow.I don't know to tackle it.

**Comment by ser888gio** [comment]
tenserflow-gpu was removed, so try without it. Source: https://pypi.org/project/tensorflow-gpu/

**Comment by Shivansh20128** [comment]
Yes, it is true that tensorflow-gpu was removed. But if you are looking to train your model with your GPU, you will need tensorflow-gpu. Since tensorflow-gpu has been removed from the recent versions of tensorflow, you need to install an older version of tensorflow, i.e., version 2.8. I faced the same problem and fixed it with version 2.8.
